### PR TITLE
Victor Kalinkin css-four-column-layout (initial submission)

### PR DESCRIPTION
**Pull request by vkalinkin**
css 4 columns

### CODE REVIEW

File: css-four-column-layout/index.html [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  <body>
+    <div class="container">
+
+      <div class="row">
+        <h2>How It Works</h2>
+      </div>
+      <div class="row">
+        <div column = "column-25-percent">
+          <img src="images/screen-1.png" alt="app switching" class="app-pic">
+          <p></p>
```
Comment (Zhiwutian) [inline css-four-column-layout/index.html:18]: Make sure to remove these empty elements. if you need the margin, some bottom margin on the above element can take care of that for you!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small change needed.

**Comment by vkalinkin** [comment]
css 4 columns 2

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**